Tasks/Popko/Task8/bl.py

```diff
@@ -0,0 +1,69 @@
+import data
+
+def get_position(number):
+    if number == 0:
+        fender, gibson, ibanez, jackson, yamaha, gretsch = data.get_position()
```

Comment:
cool variable names) but next time consider to pack it into some structure. at least a list